tapir/coop/views/shareowner.py

```diff
@@ -786,6 +793,37 @@ def is_shift_partner_of_filter(
         return queryset.filter(
             user__shift_user_data__shift_partner_of__isnull=not value
         )
+    
+    @staticmethod
+    def filter_by_join_date(queryset: ShareOwner.ShareOwnerQuerySet, name, value):
+        if value.start and value.stop:
+            from django.db.models import Min, OuterRef, Subquery
```

Comment:
Do we need those imported locally? If not, let's put them at the top of the file with the other imports.